Disable transmogrification of URDF files into SDFormat files.
Drake has its own URDF parser that we should always use.

diff --git a/src/parser.cc b/src/parser.cc
index 444d60a2..432fe42b 100644
--- src/parser.cc
+++ src/parser.cc
@@ -45,7 +45,7 @@
 #include "ScopedGraph.hh"
 #include "Utils.hh"
 #include "parser_private.hh"
-#include "parser_urdf.hh"
+// #include "parser_urdf.hh"

 namespace sdf
 {
@@ -787,6 +787,7 @@ bool readFileInternal(const std::string &_filename, const bool _convert,
   }
   else
   {
+#if 0  // Disable URDF2SDF for Drake.
     tinyxml2::XMLElement *robotXml = xmlDoc.FirstChildElement("robot");
     if (robotXml)
     {
@@ -810,6 +811,7 @@ bool readFileInternal(const std::string &_filename, const bool _convert,
     {
       sdferr << "XML does not seem to be an SDFormat or an URDF file.\n";
     }
+#endif
   }

   return false;
@@ -875,6 +877,7 @@ bool readStringInternal(const std::string &_xmlString, const bool _convert,
   }
   else
   {
+#if 0  // Disable URDF2SDF for Drake.
     tinyxml2::XMLElement *robotXml = xmlDoc.FirstChildElement("robot");
     if (robotXml)
     {
@@ -900,6 +903,7 @@ bool readStringInternal(const std::string &_xmlString, const bool _convert,
       sdferr << "XML does not seem to be an SDFormat or an URDF string.\n";
       return false;
     }
+#endif
   }
   return false;
 }
